### PR TITLE
adds 2 new events

### PR DESCRIPTION
**Pull request by AKCodez**

### CODE REVIEW

File: community/events.page.tsx [hunk 1]
```diff
@@ -952,6 +952,28 @@ const events = [
...
+    link: "https://xrpl.at/Eventbrite-FutureverseDeveloperTraining",
+    location: "Paris, France",
+    date: "November 4 - 5, 2024",
+    image:  require('../static/img/events/dev-training.png'),
+    end_date: "November 5, 2024",
+  },
+  {
+    name: "XRPL Hackathon Paris",
+    description:
+      "Join the hackathon organized by Chair UNESCO ITEN, XRPL Commons,  and KRYPTOSPHERE, for a unique opportunity to dive into the rapidly growing  open-source ecosystem. Tackle real-world challenges and create innovative  solutions that can make a meaningful impact on today's most pressing  societal issues!",
```
Comment (DennisDawson) [inline community/events.page.tsx]: "organized by Chair UNESCO ITEN" doesn't sound right to me. Be sure that's what you mean to say.

**Review by DennisDawson — APPROVED** [review]
There are several places where there are double-spaces, but they should be trimmed automatically when the page is published.

File: community/events.page.tsx (hunk 1 above)
Comment (amarantha-k) [inline community/events.page.tsx]:
```suggestion
      "Join the hackathon organized by Chaire UNESCO ITEN, XRPL Commons,  and KRYPTOSPHERE, for a unique opportunity to dive into the rapidly growing  open-source ecosystem. Tackle real-world challenges and create innovative  solutions that can make a meaningful impact on today's most pressing  societal issues!",
```

Comment (amarantha-k) [reply community/events.page.tsx]: Suggestion based on https://www.fmsh.fr/en/projects/chaire-unesco-iten.

**Pull request merged**